src/Eshopworld.WorkerProcess/IWorkerLease.cs

```diff
@@ -9,6 +9,12 @@ public interface IWorkerLease
         /// The instance id
         /// </summary>
         Guid InstanceId { get; }
+
+        /// <summary>
+        /// Priority
+        /// </summary>
+        int Priority { get; }
```

Comment:
I think this can be avoided. Shouldn't the priority belong to a worker process and not the lease?

Reply:
I would also have similar question - we did not need it before - any reason we will need it now?

Reply:
the only reason why this was introduced is that integration tests only use workerprocess, and they never make assertion for priority. Any suggestion , on how this could be done otherwise without a heavy restructuring of the way the tests are conceived?

Reply:
Also the new Priority property is readonly

Reply:
removed priority from workerlease. changed integrationtests to have a workerleasesMap where each created workerLease is mapped to its priority